sigs.yaml

```diff
@@ -2865,6 +2865,52 @@ workinggroups:
     liaison:
       github: liggitt
       name: Jordan Liggitt
+- dir: wg-batch
+  name: Batch
+  mission_statement: >
+    Discuss and enhance the support for Batch (eg. HPC, AI/ML, data analytics, CI)
+    workloads in core Kubernetes. We want to unify the way users deploy batch workloads
+    to improve portability and to simplify supportability for Kubernetes providers.
+
+  charter_link: charter.md
+  stakeholder_sigs:
+  - Apps
+  - Autoscaling
+  - Node
+  - Scheduling
+  label: batch
+  leadership:
+    chairs:
+    - github: Huang-Wei
+      name: Wei Huang
+      company: Apple
+    - github: ahg-g
+      name: Abdullah Gharaibeh
+      company: Google
+    - github: endocrimes
+      name: Danielle Lancashire
+      company: VMware
+    - github: soltysh
+      name: Maciej Szulik
+      company: Red Hat
+    - github: swatisehgal
+      name: Swati Sehgal
+      company: Intel
+  meetings:
+  - description: Regular Meeting
+    day: TBD
+    time: TBD
+    tz: UTC
+    frequency: biweekly
+    url: TBD
+    archive_url: TBD
+    recordings_url: TBD
+  contact:
+    slack: wg-batch
+    mailing_list: TBD
+    liaison:
+      github: TBD
+      name: TBD
```

Comment:
discussed in steering... probably one of @mrbobbytables (liaison for sig-apps), @tpepper (liaison for sig-node and sig-autoscaling), @liggitt (liaison for sig-scheduling)

do the wg organizers have a sense for which of the three would be most helpful?

Reply:
probably @mrbobbytables since he has experience with batch from his past life?

Reply:
I'm good with that 👍  I work with groups on the CNCF side as well.

Reply:
Added. With that, I believe we should be ready to merge.